js/src/cli/item.ts

```diff
@@ -26,7 +26,6 @@ import {
   getMetadata,
 } from "../utils/pda";
 import { InheritanceState, PermissivenessType } from "../state/common";
-import { overArgs } from "lodash";
 
 programCommand("create_item_class")
```

Comment:
At some point, not for this PR, we'll want to update this whole CLI file to use sol-command. Something to keep in mind for a later date or if larger changes are needed on the CLI before then. https://github.com/raindrops-protocol/raindrops/pull/19/files#diff-098204a6c1a323caf6d9e9831ec051cf410cb5b7cb40dd49927327f11320421b is an example you can look to if it becomes necessary.